tests/HttpClientInterception.Tests/HttpRequestInterceptionBuilderTests.cs

```diff
@@ -557,6 +557,43 @@ public static class HttpRequestInterceptionBuilderTests
             (await HttpAssert.GetAsync(options, "http://something.com")).ShouldBeEmpty();
         }
 
+        [Fact]
+        public static async Task Register_Builds_Ignore_Query_Key()
+        {
+            // Arrange
+            var options = new HttpClientInterceptorOptions();
+
+            HttpRequestInterceptionBuilder builder = new HttpRequestInterceptionBuilder()
+                .ForHost("something.com")
+                .IgnoringQuery();
+
+            // Act
+            options.Register(builder);
+
+            // Assert
+            (await HttpAssert.GetAsync(options, "http://something.com?query=1234")).ShouldBeEmpty();
+        }
+
+        [Fact]
+        public static async Task Register_Builds_Without_Ignore_Query_Key()
+        {
+            // Arrange
+            var options = new HttpClientInterceptorOptions();
+
+            HttpRequestInterceptionBuilder builder = new HttpRequestInterceptionBuilder()
+                .ForHost("something.com")
+                .IgnoringQuery()
+                .IgnoringQuery(false)
+                ;
```

Comment:
';' on the line above please.

Reply:
I like to do this on fluent interfaces as it means any subsequent addition only shows itself in the diff, rather than the previous last item as well.  Will remove though 😄